Test-Scenarios/2023-07-01/fhir_dump.json

```diff
@@ -705,146 +705,31 @@
     "featureFile" : "Sonstige_Versicherungsart",
     "requirements" : [ "A_22222" ]
   }, {
-    "id" : "a8891b03-5406-4bf5-8e16-f732479fb16c",
+    "id" : "bc1923f2-e84f-43d4-87b3-dabfc19ba9a7",
     "mainActor" : "Versicherter",
     "name" : "E-Rezept per alternative Zuweisung einlösen",
     "tags" : [ "@PRODUKT:eRp_FD", "@Funktionalität:AlternativeZuweisung", "@Hauptdarsteller:Versicherter", "@TCID:ERP_EE_ALTERNATIVE_ZUWEISUNG_01", "@TESTFALL:positiv" ],
     "description" : "Der Arzt verschreibt der GKV-Versicherten ein E-Rezept für ein apothekenpflichtiges Rezept.\n  Der Versicherte weist das E-Rezept seiner Apotheke über den Apothekendienstleister zu.",
     "feature" : "Alternative Zuweisung",
     "featureFile" : "alternative_zuweisung/alternative_Zuweisung"
   }, {
-    "id" : "5a5bfc28-5d02-40f9-ae41-1ccbb654b44a",
+    "id" : "22c4a852-12d2-4088-89c8-e431b849311c",
     "mainActor" : "Versicherter",
     "name" : "E-Rezept per alternative Zuweisung einlösen",
     "tags" : [ "@PRODUKT:eRp_FD", "@Funktionalität:AlternativeZuweisung", "@Hauptdarsteller:Versicherter", "@TCID:ERP_EE_ALTERNATIVE_ZUWEISUNG_01", "@TESTFALL:positiv" ],
     "description" : "Der Arzt verschreibt der GKV-Versicherten ein E-Rezept für ein apothekenpflichtiges Rezept.\n  Der Versicherte weist das E-Rezept seiner Apotheke über den Apothekendienstleister zu.",
     "feature" : "Alternative Zuweisung",
     "featureFile" : "alternative_zuweisung/alternative_Zuweisung"
   }, {
-    "id" : "3bd568f7-bb61-4e9f-a37f-12ab77243c4c",
+    "id" : "adc561ad-3131-4543-a31b-d8c5dc7e64e6",
     "mainActor" : "Versicherter",
     "name" : "E-Rezept per alternative Zuweisung einlösen",
     "tags" : [ "@PRODUKT:eRp_FD", "@Funktionalität:AlternativeZuweisung", "@Hauptdarsteller:Versicherter", "@TCID:ERP_EE_ALTERNATIVE_ZUWEISUNG_01", "@TESTFALL:positiv" ],
     "description" : "Der Arzt verschreibt der GKV-Versicherten ein E-Rezept für ein apothekenpflichtiges Rezept.\n  Der Versicherte weist das E-Rezept seiner Apotheke über den Apothekendienstleister zu.",
     "feature" : "Alternative Zuweisung",
     "featureFile" : "alternative_zuweisung/alternative_Zuweisung"
   }, {
-    "id" : "e7289aee-495c-4a38-a9cb-02ea3a3e34dd",
```

Comment:
Hier sind ganz viele Testfälle verschwunden. 
Muss hier bei Jenkins ein Toggle gesetzt werden, dass die eGK in der Apotheke Fälle auch mit erzeugt werden? Kannst du das prüfen?

Reply:
Die eGK Testfälle waren ausgelagert. Ich habe nachgefragt wo ich die finden kann, und werde es aktualisieren wenn ich die auch bekommen hat.